### PR TITLE
Replace piwik with matomo analytic templates

### PR DESCRIPTION
**Pull request by fnagel**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Related issues/PRs | https://github.com/sulu/sulu/discussions/7692
| License | MIT

#### What's in this PR?

Updated the Matomo templates.

#### Why?

Because the current ones are outdated.

### CODE REVIEW

**Comment by alexander-schranz** [comment]
@fnagel thx for the pull request can you add a short line to the `UPGRADE.md`

Something similar to:

```
### Piwik replaced with Matomo script

The script provided by Sulu for the piwik implementation has been updated to use mataomo path so the script is now pointing to matomo.php instead of the piwik.php file.
```

**Comment by fnagel** [comment]
@alexander-schranz Changed the `UPGRADE.md`.

**Review by alexander-schranz — APPROVED** [review]

**Comment by alexander-schranz** [comment]
@fnagel awesome. Thank you!

File: src/Sulu/Bundle/WebsiteBundle/Resources/views/Analytics/matomo/head-close.html.twig [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <script>
-    var _paq = _paq || [];
+    var _paq = window._paq = window._paq || [];
     _paq.push(['trackPageView']);
     _paq.push(['enableLinkTracking']);
     (function() {
         var u="{{ analytics.content.url|trim('/') }}";
-        _paq.push(['setTrackerUrl', u+'/piwik.php']);
+        _paq.push(['setTrackerUrl', u+'matomo.php']);
```
Comment (benr77) [inline src/Sulu/Bundle/WebsiteBundle/Resources/views/Analytics/matomo/head-close.html.twig:7]: The forward slash has been removed here, which is causing invalid URLs to be constructed.

File: src/Sulu/Bundle/WebsiteBundle/Resources/views/Analytics/matomo/body-open.html.twig [hunk 2]
```diff
@@ -1 +1 @@
-<noscript><p><img src="{{ analytics.content.url }}/piwik.php?idsite={{ analytics.content.siteId }}" style="border:0;" alt="" /></p></noscript>
+<noscript><p><img referrerpolicy="no-referrer-when-downgrade" src="{{ analytics.content.url }}/matomo.php?idsite={{ analytics.content.siteId }}&amp;rec=1" style="border:0;" alt="" /></p></noscript>
```
Comment (benr77) [inline src/Sulu/Bundle/WebsiteBundle/Resources/views/Analytics/matomo/body-open.html.twig:1]: This also needs the `|trim('/')` filter to ensure there is not a duplicate forward slash.

File: src/Sulu/Bundle/WebsiteBundle/Resources/views/Analytics/matomo/head-close.html.twig (hunk 1 above)
Comment (fnagel) [reply src/Sulu/Bundle/WebsiteBundle/Resources/views/Analytics/matomo/head-close.html.twig:7]: You are right. I copied the latest code from the Matomo docs without checking what the u variable will contain. Thanks!

File: src/Sulu/Bundle/WebsiteBundle/Resources/views/Analytics/matomo/body-open.html.twig (hunk 2 above)
Comment (fnagel) [reply src/Sulu/Bundle/WebsiteBundle/Resources/views/Analytics/matomo/body-open.html.twig:1]: You're right. This seems to be an issue even before my changes.

**Pull request merged**